.babelrc

```diff
@@ -2,11 +2,33 @@
   "presets": [
     "@babel/typescript",
     "@babel/react",
-    "@babel/stage-3",
-    ["@babel/env", {
-      "targets": {
-        "browsers": ["last 2 versions"]
+    [
+      "@babel/env",
+      {
+        "targets": {
+          "browsers": [
+            "last 2 versions"
+          ]
+        }
       }
-    }]
+    ]
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    [
+      "module-resolver",
+      {
+        "extensions": [
+          ".ts",
+          ".tsx"
+        ],
+        "root": [
+          "./src"
+        ],
+        "alias": {
+          "palette": "./src/index"
```

Comment:
When working around the project `palette` refers to the lib itself, as if you were importing from `@artsy/palette`:

```jsx
import { Box, Theme, Foo, Bar } from 'palette'
```